AlertSender.ps1

```diff
@@ -354,6 +354,17 @@ Switch ($Config.services) {
 	}
 }
 
+# Clean up old log files if configured
+if ($Config.log_expiry_days -ne 0) {
+	if ((Get-ChildItem "$PSScriptRoot\log" | Where-Object { $_.CreationTime -lt (Get-Date).AddDays(-$Config.log_expiry_days)}).Count -ne 0) {
+		Write-LogMessage -Tag 'INFO' -Message "Found $FilesToRotate log files to rotate."
+		Get-ChildItem "$PSScriptRoot\log" | Where-Object { $_.CreationTime -lt (Get-Date).AddDays(-$Config.log_expiry_days)} | Remove-Item
```

Comment:
I know you said you didn't think there needs to be an extra variable, but the statements can be simplified and made much more readable by setting a variable for the results of the log query and referencing that thereafter.



```suggestion
    $oldLogs = Get-ChildItem "$PSScriptRoot\log" | Where-Object { $_.CreationTime -lt (Get-Date).AddDays(-$Config.log_expiry_days) }

	if ($oldLogs.Count -ne 0) {
		Write-LogMessage -Tag 'INFO' -Message "Found $FilesToRotate log files to rotate."
		$oldLogs | Remove-Item
```